src/patternfly/components/NavSystem/styles.scss

```diff
@@ -0,0 +1,573 @@
+  --pf-c-nav__list-tertiary-link--PaddingRight:                         var(--pf-global--spacer--lg);
+  --pf-c-nav__list-tertiary-link--PaddingBottom:                        var(--pf-global--spacer--md);
+  --pf-c-nav__list-tertiary-link--PaddingLeft:                          var(--pf-global--spacer--lg);
+  
+  // Nav list tertiary link color
+  --pf-c-nav__list-tertiary-link--Color:                                var(--pf-global--Color--dark-100);
+  --pf-c-nav__list-tertiary-link--hover--Color:                         var(--pf-global--link--Color--active);
+  --pf-c-nav__list-tertiary-link--focus--Color:                         var(--pf-global--link--Color--active);
+  --pf-c-nav__list-tertiary-link--active--Color:                        var(--pf-global--link--Color--active);
+  --pf-c-nav__list-tertiary-link--current--Color:                       var(--pf-global--link--Color--active);
+  --pf-c-nav__list-tertiary-link--disabled--Color:                      var(--pf-global--link--Color--disabled);
+
+  // Nav list tertiary link background color
+  --pf-c-nav__list-tertiary-link--BackgroundColor:                      transparent;
+
+  // Nav section
+  --pf-c-nav__section--MarginTop:                                       var(--pf-global--spacer--xl);
+
+  // Nav section title
+  --pf-c-nav__section-title--PaddingTop:                                var(--pf-global--spacer--sm);
+  --pf-c-nav__section-title--PaddingRight:                              var(--pf-global--spacer--2xl);
+  --pf-c-nav__section-title--PaddingBottom:                             var(--pf-global--spacer--sm);
+  --pf-c-nav__section-title--PaddingLeft:                               var(--pf-global--spacer--2xl);
+  --pf-c-nav__section-title--FontSize:                                  var(--pf-global--FontSize--sm);
+  --pf-c-nav__section-title--Color:                                     var(--pf-global--Color--dark-200);
+  --pf-c-nav__section-title--FontWeight:                                var(--pf-global--FontWeight--semi-bold);
+
+  /* stylelint-enable */
+
+  // Explicitly hide menu for IE
+  [hidden] {
+    display: none;
+  }
+
+  // Temporary
+  .pf-u-sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border: 0;
+  }
+
+  // If navigation is in header (horizontal nav), remove padding
+  .pf-l-page__header-nav & {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  padding-top: var(--pf-c-nav--MarginTop);
+  padding-bottom: var(--pf-c-nav--MarginBottom);
+
+  // List - default
+  &__list {
+
+    // Item
+    &-item {
+      position: relative;
+      display: flex;
+      flex-wrap: wrap;
+      align-items: baseline;
+      background-color: var(--pf-c-nav__item--BackgroundColor);
+    }
+
+    // Link
+    &-link {
+      position: relative;
+      display: flex;
+      flex: 1;
+      align-items: baseline;
+      padding:
```

Comment:
Looks like there are a few instances of shorthand `padding` - per the contribution docs, these shouldn't be shorthand - https://deploy-preview-503--pf-next.netlify.com/guidelines/#shorthand-notation